### DISCUSSION TITLE
v2.15.0

### DISCUSSION THREAD

**Discussion by jgraichen** [jgraichen/redmine_dashboard#437] [Announcements]
### New

- Support for Redmine 5.1 by @jgraichen

### Fixes

- Workaround a change in Redmine 5.1 to ignore roles with only the `:edit_own_issues` permission by @jgraichen

**Full Changelog**: https://github.com/jgraichen/redmine_dashboard/compare/v2.14.0...v2.15.0

---

### Looking for testers!

Unfortunately, I do not have access to any larger Redmine project anymore, nor am I using Redmine anywhere anymore. I can only test with some artificially constructed local project that cannot represent any real project or usage. I would really appreciate it if you can test v2.15.0 in your project or organization and report any findings (or their absence). Thank you!

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraichen/redmine_dashboard/releases/tag/v2.15.0'>v2.15.0</a>.</em>

**Comment by fedeyobe** [comment]
Thanks for this! i have a question, is there any way to see the parent task of an issue?

thanks again!!
